game/scripts/npc/abilities/silencer_glaives_of_wisdom.txt

```diff
@@ -12,7 +12,7 @@
     "AbilityUnitTargetTeam"                               "DOTA_UNIT_TARGET_TEAM_ENEMY"
     "AbilityUnitTargetType"                               "DOTA_UNIT_TARGET_HERO | DOTA_UNIT_TARGET_BASIC"
     "SpellImmunityType"                                   "SPELL_IMMUNITY_ENEMIES_NO"
-    "AbilityUnitDamageType"                               "DAMAGE_TYPE_PHYSICAL"
```

Comment:
I don't think the damage nerfs are enough to make up for changing the damage back from physical to pure.